CHANGELOG.md

```diff
@@ -16,6 +16,33 @@ package updates, you can specify your package dependency using
 
 ---
 
+## [0.0.4] - 2021-03-29
+
+### Additions
+
+More new algorithms to join the party:
+
+- A lazy version of the standard library's two `split` methods. ([#78])
+- `firstNonNil(_:)` returns the first non-`nil` element from an
+  optional-generating transform. ([#31])
+- `uniquePermutations()` skips duplicates when generating permutations of a
+  collection. ([#91])
+- The `reductions` methods return all the in-between states of reducing a
+  sequence or collection. ([#46])
+
+### Fixes
+
+- Methods and computed properties are more consistently marked as inlinable, 
+  resolving a performance regression. 
+- The `Stride` type now efficiently calculates distances between positions,
+  supported by the underlying collection.
+- Better test coverage and improved diagnostics for comparing sequences.
+- Fixed links and improved documentation.
+
+The 0.0.3 release includes contributions from [bjhomer], [danielctull],
```

Comment:
Hey @natecook1000 :) 
Should this be 0.0.4?

Reply:
Gah, thanks!